packages/node-sdk/paima-db/src/paima-tables.ts

```diff
@@ -753,6 +753,61 @@ const TABLE_DATA_CDE_DYNAMIC_PRIMITIVE_CONFIG: TableData = {
   creationQuery: QUERY_CREATE_TABLE_CDE_DYNAMIC_PRIMITIVE_CONFIG,
 };
 
+const QUERY_CREATE_TABLE_EVENT = `
+CREATE TABLE event (
+  id SERIAL PRIMARY KEY,
```

Comment:
We shouldn't use `serial` fields here. The STF will eventually process inputs in parallel, making the order non-deterministic

Reply:
I can probably change the pk to block height + index inside the block instead, but I'm not sure it would make any difference. Can you explain what does it mean to process inputs in parallel? Because even then you could call the stf multiple times in parallel and then serialize the results.

Reply:
I think having the ID be deterministic even in the case where transactions are executed in parallel is equivalent to the other open discussion on this PR: https://github.com/PaimaStudios/paima-engine/pull/409#discussion_r1728245329